templates/organization/create-composition.html

```diff
@@ -8,19 +8,25 @@
     <h2>{% trans "Add relationship" %}</h2>
     <div class="row relationship_type-row field-bg bg-info">
         <div class="col-sm-12">
-            {% if form.name.errors %}
+            {% if form.parent.data == organization.id|stringformat:"i" and form.parent.errors %}
+                <div class="form-group has-error has-feedback">
+            {% elif form.child.data == organization.id|stringformat:"i" and form.child.errors %}
```

Comment:
I updated some of the form error display logic for this view. This was because I was trying and failing to create a Composition for testing purposes, when I realized that the form errors were being displayed improperly.

The basic problem is there are two inputs, `parent` and `child`, whose `name` attributes are assigned dynamically, and instead of figuring out which errors to display for which input we were instead using the values `name` and `relationship_type` to determine the presence of errors, neither of which refers to the fields in question.